guides/Circle CI 2.md

```diff
@@ -0,0 +1,195 @@
+# Circle CI 2.0 configurations for Rails project
+
+## Basic Concepts
+
+CircleCI 2.0 operates on a differently structurized YAML file located under `.circleci/config.yml`.
+
+It's safe to say that every config should contain at least three keywords: `version, jobs`.
```

Comment:
Each config must contain the following 3 keys: `version` and `jobs`.

Initially, your setup will look like this: